package-lock.json

```diff
@@ -9338,6 +9256,18 @@
         "node": "*"
       }
     },
+    "node_modules/ast-types": {
+      "version": "0.13.4",
+      "resolved": "https://nexus3.topdesk.com/nexus/repository/npm/ast-types/-/ast-types-0.13.4.tgz",
```

Comment:
Looks like you got your npm config set to a custom registry

Reply:
Good catch, I completely overlooked this, thanks! I re-created the package-lock.json